helpers/addresses.py

```diff
@@ -264,10 +265,11 @@
         "native.cvx": "0xBCee2c6CfA7A4e29892c3665f464Be5536F16D95",
         "native.mstableImBtc": "0xd409C506742b7f76f164909025Ab29A47e06d30A",
         "native.mstableFpMbtcHbtc": "0x54D06A0E1cE55a7a60Ee175AbCeaC7e363f603f3",
-        "native.vestedCVX": "0x3ff634ce65cDb8CC0D569D6d1697c41aa666cEA9",
+        "native.vestedCVX": "0x898111d1F4eB55025D0036568212425EE2274082",
         "native.bbveCVX-CVX-f": "0x98Ca7AFa876f0e15494E76E92C5b3658cdE1Ffe1",
         "native.bcrvIbBTC": "0x6D4BA00Fd7BB73b5aa5b3D6180c6f1B0c89f70D1",
         "native.remDigg": "0x4055D395361E73530D43c9D4F18b0668fe4B5b91",
+        "native.badgerCrv": "0x1905FD2D2D09792eE058C2b46a05F11630a1EcA1"
```

Comment:
confirmed is deployed by badger dao, verified on etherscan and `want` is `badgerWBTC_f` token.